targets/tofino/base/expression_resolver.cpp

```diff
@@ -481,7 +482,12 @@ const ExternMethodImpls EXTERN_METHOD_IMPLS(
           }
           auto &applyStepper = FlayTarget::getStepper(externInfo.programInfo.get(), state);
           applyDecl->body->apply(applyStepper);
-          return state.get(paramRefs.at(1));
+          if (applyParameters->size() == 2) {
+              return state.get(paramRefs.at(1));
+          }
+          return static_cast<const IR::Expression *>(ToolsVariables::getSymbolicVariable(
```

Comment:
For the apply function with only one parameter, it doesn't return anything, can we just return nullptr (instead of create a symbolic value) after handling the apply call if applyParameters->size() == 1?

Reply:
Ah yeah, you are right.